validator/accounts/accounts_delete.go

```diff
@@ -101,7 +101,10 @@ func DeleteAccountCli(cliCtx *cli.Context) error {
 	}); err != nil {
 		return err
 	}
-	log.WithField("publicKeys", allAccountStr).Info("Accounts deleted")
+	log.WithField("publicKeys", allAccountStr).Warn(
+		"Attempted to delete accounts. IMPORTANT: please run validator accounts list to ensure" +
```

Comment:
Can you wrap `validator accounts list` in backtics/apostrophes to distinguish the command from the rest of the text?